lib/std/js/jsasynchttpclient.nim

```diff
@@ -0,0 +1,86 @@
+## `Async <asyncjs.html>`_ `HttpClient <httpclient.html>`_ for JavaScript implemented on top of `jsfetch <jsfetch.html>`_
+##
+## .. Note:: jsasynchttpclient module requires `-d:nimExperimentalJsfetch`
+when not defined(js):
+  {.fatal: "Module jsasynchttpclient is designed to be used with the JavaScript backend.".}
+
+import std/[jsfetch, asyncjs]
+from std/uri import Uri
+
+type JsAsyncHttpClient* = ref object of JsRoot
+
+func newJsAsyncHttpClient*(): JsAsyncHttpClient = discard
```

Comment:
I don't understand how this could work (isn't there some imporjs/importc missing?)

Reply:
It just constructs and returns the type.